package.json

```diff
@@ -49,8 +49,27 @@
     "nps": "^5.7.1",
     "nps-utils": "^1.3.0",
     "prettier-eslint-cli": "^8.0.0",
+    "prettier-plugin-svelte": "^3.1.2",
     "rimraf": "^5.0.5",
-    "strip-indent": "^3.0.0"
+    "strip-indent": "^3.0.0",
+    "svelte": "^4.2.9",
+    "svelte-eslint-parser": "^0.33.1"
+  },
+  "peerDependencies": {
+    "prettier-plugin-svelte": "^3.1.2",
+    "svelte": "^4.2.9",
```

Comment:
> (svelte should not be peer.)

`vue-eslint-parser` should also be marked as peer.

Reply:
@JounQin I agree, but that has nothing to do with this pull request. Do you want me to still change it?

Reply:
Oh, sorry, lets keep it as-is for now.

Reply:
Again.

> (svelte should not be peer.)

Please remove it from peer. (We didn't rely on it directly.)